src/app/components/critical-apparatus/critical-apparatus.component.html

```diff
@@ -0,0 +1,6 @@
+<ng-container *ngFor="let element of entries"> 
+    <evt-apparatus-entry-detail *ngIf="element?.lemma && true" [data]="element" (click)="stopPropagation($event)" class="appa-list">
```

Comment:
Change "appa-list" in "app-list"

Reply:
In general I felt 'app' easy to be confused with app(lication) instead of app(aratus). That's why I used sometimes "appa" but I'm changing it to 'app' accordingly to the revision